Guidelines: Expand on "deprecations are not for cleanup" (#1357)
openstreetmap/id-tagging-schema: GUIDELINES.md

- ➕ **Suggested Additions**: Presets can suggest additional tags. These suggestions must be clearly supported by the wiki and community consensus.
- 🔄 **Updates**: Deprecation rules can suggest updating tags. Good documentation and consensus are needed for these deprecations.

**In both cases, _indicators for consensus_ are:**

- The deprecation is documented in the wiki and is either official (resulting from a proposal process) or long-standing (about a year).
- There is a significant drop in usage compared to previous numbers, with a negative trend ([visible in the graph](https://taghistory.raifer.tech/)).
- Usage of the deprecated tag remains stagnant for a longer period (about a year).

**Deprecations are not for cleanup:**

Deprecation rules work such that the user sees a message with suggestions and can act only when editing the given element. This makes them well-suited for gradual, human-reviewed updates of taggings like crossings. However, they are not suitable for cleaning up incorrect tagging from the database, especially for low-volume changes.

> There are, however, alternatives to consider: 

Reviewer: ```suggestion
There are, however, alternatives to consider:
- You can simply resurvey objects in your area, sometimes mapping based on aerial imagery and other resources available for OpenStreetMap mapping
- You can contact mappers (via changeset comments) and local communities (by creating notes, [writing on local community forums](https://wiki.openstreetmap.org/wiki/Contact_channels) and suggest looking at their local areas
```

Reviewer: maybe also note that checking area distribution and sample of edits may be a good idea? sometimes it is a single edit or single mapper

sometimes it is even revertable as bogus import or vandalism

Author: I agree those are good things to do. But the cases when I link to this guideline to close issues or ask for clarification seem to not fit into this category of local or single user edits.

Maybe we should add a wiki page or a section on the organized editing guidlines to go into more details? This question of "how to cleanup values" should be common outside of this repo, right?

Reviewer: > But the cases when I link to this guideline to close issues or ask for clarification seem to not fit into this category of local or single user edits.

ok, I was unaware of that, then it makes no sense

> Maybe we should add a wiki page or a section on the organized editing guidlines to go into more details? This question of "how to cleanup values" should be common outside of this repo, right?

+1 though I have no ready link - but if that case does not exist at problem at this repo we may simply not mention it

> - Your cleanup task might be eligible for an automated (bot) edit. [Please learn more on the wiki…](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct)

Reviewer: ```suggestion
- Your cleanup task might be eligible for an automated (bot) edit. [Please learn more on the wiki…](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct) If mass-replace, using script or JOSM/level0 is beyond your abilities you can still suggest such edit at [OSM forums](https://community.openstreetmap.org/)
```

Author: Good idea, I added it to the end of the list separately.

> - If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick.

Reviewer: maybe it should be made clear that you should consult community? and mass changing tag in JOSM/level0 is still an automated edit?

Reviewer: ```suggestion
- If your automated edit is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick - just remember to ask other mappers first, as required for such edits.
```

Author: Thanks, I committed a modified version of this.

> - If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick.

Author: ```suggestion
- If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick. This is still an automated edit, though, so the [Guidelines apply](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct). Please talk to other mappers first.
```

> - If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick. This is still an automated edit, though, so the [Guidelines apply](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct). Please talk to other mappers first.

Reviewer: ```suggestion
- If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick. Mass-replace without checking every single object is still an automated edit, though, so the [Guidelines apply](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct). Please talk to other mappers first.
```

Author: Good addition. ChatGPT version is…

```suggestion
- If your task is small enough, a few [editing sessions in JOSM](https://wiki.openstreetmap.org/wiki/JOSM) will often do the trick. However, mass-replacing without checking each object is still considered an automated edit, so the [guidelines apply](https://wiki.openstreetmap.org/wiki/Automated_Edits_code_of_conduct). Please consult other mappers first.
```

> - A good way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/).

Reviewer: ```suggestion
- A way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/), but someone creating such challenge should provide good instruction, ensure that mappers are not editing blindly and repair any damage caused by it. Consider making challenge private and sharing it in controlled fashion.
```

I would consider dropping it altogether. Or explicitly warn that some people will go through random MR challenges blindly editing without actual verification or thinking.

Author: No, I think it is the responsibility of MR to guide users to using the tool right. The role of this section is only to give starting points for further steps elsewhere…

> - A good way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/).

Author: ```suggestion
- A good way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/).

Should those options be not for you, you can always suggest such changes in the [OSM community forum](https://community.openstreetmap.org/).
```

> - A good way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/). For small changes, a [Cooperative Challenge](https://learn.maproulette.org/en-us/documentation/creating-cooperative-challenges/) will likely be a good fit, allowing mappers to change the tagging directly from MapRoulette.

Reviewer: note that it is not really a good way for anything that requires actual review but has an obvious "fix"

too many people edit blindly or use MR as an automated edit, run via blindly made human edits

Author: Agreed, the feature is not there yet and it's better to be cautious here.

> - A good way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/). For small changes, a [Cooperative Challenge](https://learn.maproulette.org/en-us/documentation/creating-cooperative-challenges/) will likely be a good fit, allowing mappers to change the tagging directly from MapRoulette.

Author: ```suggestion
- A good way to work down a list of tasks is to create [a MapRoulette Challenge](https://maproulette.org/).
```
